internal/ignition/ignition.go

```diff
@@ -225,8 +248,25 @@ func (g *installerGenerator) generateLsoManifests(ctx context.Context, installer
 	return nil
 }
 
+func (g *installerGenerator) generateOcsManifests(ctx context.Context, installerPath string, envVars []string, ocsValidatorConfig *ocs.Config) error {
+	if g.checkOcsEnabled() {
+		err := g.createManifestDirectory(installerPath, envVars)
```

Comment:
Priyanka, If OCS is enabled, doesn't that mean LSO will have to be enabled as well seeing that OCS will need to use LSO ?

Reply:
yes @nbalacha , if OCS is enabled, LSO will be automatically enabled.

Reply:
@gamli75 , we can check the creation of manifests in e2e tests as it will require `install-config manifest` , due to which I am testing only individual functions. Also there is not test for `Generate()` in `ignition.go` . Let me know your thoughts on this.